src/cmds/build/bundle.ts

```diff
@@ -102,10 +97,11 @@ async function closeBundleStream(stream, bundleString, options) {
  * @param {boolean} options.stripComments
  * @returns {string} - The postprocessed bundle string
  */
-function postProcess(bundleString, options) {
+function postProcess(bundleString: string | null, options: Option): string {
 
   if (typeof bundleString !== 'string') {
-    return null;
+    console.log(bundleString + 'is not an acceptable bundleString');
+    return "";
```

Comment:
What motivated this change? Can this function not return `string | null`?

Reply:
Basically the end function won't accept an argument that is possibly null. Another option could be https://github.com/MetaMask/snaps-cli/pull/75/commits/68f5ef1080236e101b4975675ed53735735123ea 

![Screen Shot 2021-02-01 at 3 05 28 PM](https://user-images.githubusercontent.com/58123480/106529187-ef758980-649e-11eb-81df-707ea29063aa.png)